### DISCUSSION TITLE
[1 febbraio 2024] Esercizio 6

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/psd2223#268] [Esami]
Analizzare il seguente circuito sequenziale e disegnare l’automa (macchina a stati) corrispondente.

Scrivere una sequenza di bit di ingresso e i relativi stati dell’automa che a partire dallo stato codificato 00 produca un output uguale ad 1.

<div align="center">

![image](https://github.com/user-attachments/assets/54a5b31b-9311-4180-a023-d2ea0d183f1e)

</div>

**Comment by Nw4pS** [comment]
a)= prima parte
b)= scrivere la sequenza
![Screenshot_20240909_135443_Samsung Notes.jpg](https://github.com/user-attachments/assets/ed90bb7e-3f36-4000-995c-2d36ec8aec05)
